### DISCUSSION TITLE
pyvista 3d visualization problem

### DISCUSSION THREAD

**Discussion by prab23** [gempy-project/gempy#818] [Q&A] [answered]
When I try to visualize the geo_model in 3d, I have set vertical exaggeration to 5. But along with the model it also scales up labels and make it unable to read them properly.

`g3d = gp.plot_3d(geo_model,
                 show_topography=True,
                 show_lith=True,
                 show_surfaces=True,
                 ve=5)`
                 
                 
![image](https://github.com/cgre-aachen/gempy/assets/139199787/e002cf63-05ce-4119-9c51-71cd38cac718)

**Comment by AlexanderJuestel** [comment]
Hello @prab23, 

please try the following and adjust the font size as you need. If that does not work, we will try something else ;)

```python

import pyvista as pv
pv.global_theme.font.size = 20

```

**Reply by prab23** [reply]
Sorry, It didn't work. Result is same as previous.

**Reply by AlexanderJuestel** [reply]
We are working on a fix (https://github.com/cgre-aachen/gempy/pull/820). However, there may be an issue with PyVista (see the referenced issue within the PR). We hope to have this resolved soon!

**Comment by AlexanderJuestel** [comment, marked as answer]
@prab23 Please see the comment in this issue: https://github.com/pyvista/pyvista/issues/4768#issuecomment-1679091967 and the referenced PR for that. We also hope that #820 will fix your issue from our side
